src/index.html

```diff
@@ -4,10 +4,54 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article data-qa="card" class="product">
+      <img
+        class="product__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27\0022 Retina 5K Monoblock (MNED2UA/A)"
```

Comment:
Replace the escaped double quote '\0022' with a single quote to avoid rendering issues.